webpack/assets/javascripts/react_app/redux/middlewares/ajaxRequestAction.js

```diff
@@ -0,0 +1,32 @@
+import API from '../../API';
+
+const get = (payload, url, store, consts) => {
+  store.dispatch({
+    type: consts.REQUEST,
```

Comment:
could you rename `consts` to something more clear, `actionTypes` for example

Reply:
renamed thanks!